src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClientExtensions.cs

```diff
@@ -123,15 +123,15 @@ private static IEnumerable<PSRoleAssignment> ToPSRoleAssignments(this IEnumerabl
             {
                 assignment.Properties.RoleDefinitionId = assignment.Properties.RoleDefinitionId.GuidFromFullyQualifiedId();
 #if !NETSTANDARD
-                PSADObject adObject = adObjects.SingleOrDefault(o => 
-                    o.Id == assignment.Properties.PrincipalId) ?? 
-                    new PSADObject() { Id = assignment.Properties.PrincipalId };
+				PSADObject adObject = adObjects.SingleOrDefault(o =>
+					o.Id.ToString() == assignment.Properties.PrincipalId) ??
+					new PSADObject() { Id = Guid.Parse(assignment.Properties.PrincipalId)};
```

Comment:
Why? Because in PSADObject : 'Id' is Guid type, But in the new 'Microsoft.Azure.Management.Authorization' package : 'PrincipalId' is string type. So I do the change.